packages/libs/lambda-at-edge/src/lib/copyRequiredServerFiles.ts

```diff
@@ -0,0 +1,39 @@
+import path from "path";
+import fse from "fs-extra";
+
+export const copyRequiredServerFiles = async ({
+  nextConfigDir,
+  destination
+}: {
+  nextConfigDir: string;
+  destination: string;
+}): Promise<void> => {
+  const REQUIRED_SERVER_FILES = path.join(
+    nextConfigDir,
+    ".next/required-server-files.json"
+  );
```

Comment:
Not really sure what the point of this file is, and why it's not an `.nft.json`, but seems to have required content anyway.